sdk/servicefabricmanagedclusters/Azure.ResourceManager.ServiceFabricManagedClusters/src/Generated/RestOperations/ApplicationTypeVersionsRestOperations.cs

```diff
@@ -409,7 +409,6 @@ internal HttpMessage CreateDeleteRequest(string subscriptionId, string resourceG
             uri.AppendPath(version, true);
             uri.AppendQuery("api-version", _apiVersion, true);
             request.Uri = uri;
-            request.Headers.Add("Accept", "application/json");
```

Comment:
accept header is removed for all Delete request, need to verify if this is intended.